spec/views/events/schedule.html.erb_spec.rb

```diff
@@ -2,23 +2,52 @@
 
 RSpec.describe 'events/schedule', type: :view do
 
-  before :each do
-    @league = FactoryBot.create(:league, :with_matches,
-                                deadline: Date.parse('23.12.2017'),
-                                startdate: Date.parse('24.12.2017'),
-                                enddate: Date.parse('31.12.2017'),
-                                gameday_duration: 7)
-    assign(:event, @league)
-    assign(:schedule_type, 'league')
-    assign(:matches, @league.matches)
-  end
+  describe 'page layout' do
+    before :each do
+      @league = FactoryBot.create(:league, :with_matches,
+                                  deadline: Date.parse('23.12.2017'),
+                                  startdate: Date.parse('24.12.2017'),
+                                  enddate: Date.parse('31.12.2017'),
+                                  gameday_duration: 7)
+      assign(:event, @league)
+      assign(:schedule_type, 'league')
+      assign(:matches, @league.matches)
```

Comment:
Are these variables used anywhere? `event, schedule_type, matches`

Reply:
I didn't write those lines, but removing them makes the tests red. Maybe they are used within rendering?

Reply:
The assign statements replace the controller lines:
```Ruby
@event = @league
@schedule.type = 'league'
@matches = league.matches
```
Since the controller function isn't called in view tests you have to assign these values manually to render the template. Was that the question? @usame-algan

Reply:
Yes, I was wondering what those lines were for since I didnt find them used anywhere in the file. Thanks for clarification!